OneMore/Commands/File/ImportCommand.cs

```diff
@@ -524,7 +518,7 @@ private async Task ImportMarkdown(string filepath)
 				return;
 			}
 
-			var files = Directory.GetFiles(Path.GetDirectoryName(filepath), Path.GetFileName(filepath));
+			var files = Directory.GetFiles(Path.GetDirectoryName(@filepath), Path.GetFileName(@filepath));
```

Comment:
Where is the `@filepath` variable defined? How does it differ from `filepath`?

Reply:
Looks like it's a C# language feature "verbatim identifier" [https://stackoverflow.com/a/92045](What's the use/meaning of the @ character in variable names in C#?).
So it's just `filepath` as far as I understand.

Reply:
Actually, I was looking at this hint to handle filenames with spaces: [Validating File Path w/Spaces in C#](https://stackoverflow.com/questions/1848169/validating-file-path-w-spaces-in-c-sharp). This helped to read filenames with spaces which in particular occur when files are duplicated adding spaces to the filename.

Reply:
That's not what the `@` character does. It's the [verbatim string declarator](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim). It doesn't imply anything special regarding spaces in filenames.